### PR TITLE
Create NoMach Page.

### PR DESCRIPTION
**Pull request by AnastasiiaSorina**
- [x]  design
- [x]  implementation
- [x]  ❌replace 404 text with svg/png.    _It was possible to implement the design of the page using text and CSS properties. Without using svg/png images._

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { Route, Routes } from "react-router-dom";
 import "../src/_styles/index.scss";
 import AboutMe from "./pages/AboutMe";
 import HomePage from "./pages/HomePage";
+import NoMachPage from "./pages/NoMachPage";
```
Comment (TaniaZaslon) [inline src/App.tsx]: Let's add alias for /pages rout "@pages" like it implemented for @components. https://github.com/AnastasiSo/Portfolio-Project/blob/884e034af722f265d04e3a0d9e32de916c762f5d/webpack.config.js#L40

File: src/_styles/_noMachPage.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    line-height: 300px;
+    -webkit-text-stroke-width: 2px;
+    -webkit-text-stroke-color: $dark-blue;
+  }
+  h3 {
+    @include h3-font;
+  }
+  p {
+    margin-top: 12px;
+    margin-bottom: 24px;
```
Comment (TaniaZaslon) [inline src/_styles/_noMachPage.scss]: Let's use $size-unit here

**Review by TaniaZaslon — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (AnastasiiaSorina) [reply src/App.tsx]: - [x] Done [aab8067](https://github.com/AnastasiSo/Portfolio-Project/pull/27/commits/aab806757f8426c4827d1201b10ad8aac5e12d23) and [2a41875](https://github.com/AnastasiSo/Portfolio-Project/pull/27/commits/2a41875f2ea89005e1748cfd4a6182bbda3f6af6).

File: src/_styles/_noMachPage.scss (hunk 2 above)
Comment (AnastasiiaSorina) [reply src/_styles/_noMachPage.scss]: - [x] Done [fdae40c](https://github.com/AnastasiSo/Portfolio-Project/pull/27/commits/fdae40c41792e6160694cc32968d80e655165f89).

File: src/_styles/_noMachPage.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
+.no-mach-page {
+  text-align: center;
+
+  h1 {
+    color: transparent;
```
Comment (TaniaZaslon) [inline src/_styles/_noMachPage.scss]: `color: transparent;` already stated in **h1-font-404** and will be added by this include `@include h1-font-404;` So no need to add it twice.

File: src/components/navigation/Menu.tsx [hunk 4]
```diff
@@ -5,46 +5,44 @@ import { NavHashLink } from "react-router-hash-link";
 
 // TODO: Find a solution for the behavior of contacts buttons when the checkout page is active.
```
Comment (TaniaZaslon) [inline src/components/navigation/Menu.tsx]: If this TODO is still relevant?

**Review by TaniaZaslon — CHANGES_REQUESTED** [review]

File: src/_styles/_noMachPage.scss (hunk 3 above)
Comment (AnastasiiaSorina) [reply src/_styles/_noMachPage.scss]: - [x] Done [7c57c8b](https://github.com/AnastasiSo/Portfolio-Project/pull/27/commits/7c57c8bddaa21cf6d3c9c55a5eceb99aae234219).

File: src/components/navigation/Menu.tsx (hunk 4 above)
Comment (AnastasiiaSorina) [reply src/components/navigation/Menu.tsx]: - [x] Done [873e8a0](https://github.com/AnastasiSo/Portfolio-Project/pull/27/commits/873e8a0fe0fa5738728da19d289e939fb9e09069).

**Review by TaniaZaslon — APPROVED** [review]
Looks good.

**Pull request merged**